src/hb-ot-hmtx-table.hh

```diff
@@ -146,7 +146,7 @@ struct hmtxvmtx
 
     _mtx.fini ();
 
-    if (unlikely (c->serializer->ran_out_of_room || c->serializer->in_error ()))
+    if (unlikely (c->serializer->ran_out_of_room () || c->serializer->in_error ()))
```

Comment:
Just `in_error()` should be enough.

Reply:
atm ERR_RAN_OUT_OF_ROOM is not treated as fatal, so in_error() won't return true if ran out of room is set.